weed/storage/disk_location.go

```diff
@@ -208,8 +209,14 @@ func (l *DiskLocation) concurrentLoadingVolumes(needleMapKind NeedleMapKind, con
 func (l *DiskLocation) loadExistingVolumes(needleMapKind NeedleMapKind) {
 
 	workerNum := runtime.NumCPU()
-	if workerNum <= 10 {
-		workerNum = 10
+	val, ok := os.LookupEnv("GOMAXPROCS")
+	if ok {
+		num, err := strconv.Atoi(val)
+		if err != nil || num < 1 {
+			num = 10
+			glog.Warningf("failed to set worker number from GOMAXPROCS , set to default:10")
+		}
+		workerNum = num
 	}
```

Comment:
If not found, limit to a default 10 with previous logic.

Reply:
So in this situation,  the worker number is set to 10 if "GOMAXPROCS" not found,  and never set to NumCPU()?

Reply:
> If not found, limit to a default 10 with previous logic.

So in this situation, the worker number is set to 10 if "GOMAXPROCS" not found, and never set to NumCPU()?